scripts/cd2nroff

```diff
@@ -189,7 +189,7 @@ sub single {
     # cut off the leading path from the file name, if any
     $f =~ s/^(.*[\\\/])//;
 
-    push @desc, ".\\\" generated by cd2nroff $cd2nroff from $f\n";
+    push @desc, ".\\\" generated by cd2nroff $cd2nroff\n";
```

Comment:
Two lines above it removes the path from the name, how can it be absolute?

Reply:
Oh sorry, this has changed compared to the last tested version before going to sleep. I'm deleting this.